packages/next-on-pages/src/buildApplication/processVercelFunctions/dedupeEdgeFunctions.ts

```diff
@@ -202,12 +245,19 @@ async function buildFunctionFile(
 			join(relativeImportPath, addLeadingSlash(path)),
 		);
 
-		functionImports += `import { ${keys} } from '${importPath}';\n`;
+		const namedExportsId = `getNamedExports_${chunkMapIdx++}`;
+		chunksExportsMap.set(namedExportsId, new Set([...keys.split(',')]));
```

Comment:
Suggest: replace with `.filter`to be more explicit about the intention of `[...]`

Reply:
sorry I don't follow, how are you suggesting to use `.filter`? 🙂

Reply:
I thought `[...keys.split(',')]` would filter out the empty strings but it doesn't so you can actually replace it with `keys.split(',')` only

Reply:
oh yeah.... why did I was spreading 😕

must have been because of some refactoring or something! 😓 

good catch! Thanks! ❤️